cmd/bb_worker/main.go

```diff
@@ -236,9 +236,8 @@ func main() {
 					clock.SystemClock,
 					normalizer,
 					/* defaultAttributesSetter = */ func(requested virtual.AttributesMask, attributes *virtual.Attributes) {
-						// No need to set ownership
```

Comment:
Wait. This affects the ownership on the top-level build directory. Not the per-action directory. This should be left as is. We should change pkg/builder/virtual_build_directory.go instead.